internal/controller/install/armadaserver_controller_test.go

```diff
@@ -429,12 +433,12 @@ func TestSchedulerReconciler_createIngress(t *testing.T) {
 
 	input := v1alpha1.ArmadaServer{
 		TypeMeta: metav1.TypeMeta{
-			Kind:       "Lookout",
+			Kind:       "armadaserver",
```

Comment:
typo fix?

Reply:
Yep, good catch, can you also please capitalize it? All k8s Kinds start with a capital letters.